test/parallel/test-process-hrtime.js

```diff
@@ -35,7 +35,7 @@ validateTuple(process.hrtime(tuple));
 const invalidHrtimeArgument = common.expectsError({
   code: 'ERR_INVALID_ARG_TYPE',
   type: TypeError,
-  message: 'The "process.hrtime()" argument must be of type Array'
+  message: 'The "process.hrtime()" property must be of type Array'
```

Comment:
Using the word `argument` here was correct. Please wait for a decision to #13739, but this should not be changed.